examples/catalog/test/custom_semantics_test.dart

```diff
@@ -19,11 +19,11 @@ void main() {
 
     // Verify it correctly exposes its semantics.
     // TODO(goderbauer): Use `SemanticsTester` after https://github.com/flutter/flutter/issues/12286.
-    final SemanticsNode semantics = tester
-        .renderObject(find.byType(AdjustableDropdownListTile))
-        .debugSemantics;
+    final SemanticsNode semantics =
```

Comment:
Ideally this would be closer to the original:
```
    final SemanticsNode semantics = tester
      .renderObject(find.byType(AdjustableDropdownListTile))
      .debugSemantics;
```

Reply:
with line length 110 it is now
+    final SemanticsNode semantics =
+        tester.renderObject(find.byType(AdjustableDropdownListTile)).debugSemantics;

Reply:
In this case the format the developer hand-crafted for this complex expression is better than the automatically formatted one.

Is there some way to recognize this as a legitimate way to format expression:
```
    final SemanticsNode semantics = tester
      .renderObject(find.byType(AdjustableDropdownListTile))
      .debugSemantics;
```

and not transform it to this:
```
    final SemanticsNode semantics =
        tester.renderObject(find.byType(AdjustableDropdownListTile)).debugSemantics;
```

Reply:
There is one way to do that currently but it isn't great.
```dart
    final SemanticsNode semantics = tester //
      .renderObject(find.byType(AdjustableDropdownListTile))
      .debugSemantics;
```

I've added a tweak to the formatter to not care about keeping the whole expression on one line as that matches flutter style a bit better. Good news the diff at least for these examples is small so it wouldn't be a prohibitive tweak to upstream.

Please review this diff and let me know if it is overall positive.
https://github.com/flutter/flutter/pull/25251/commits/1a32ac8f9dab317407b2d2b7d9f9bc5e750431c7